File: problems/2502-sort-the-people/sort-the-people.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        merged_tuples = [(names[i], heights[i]) for i in range(0, len(names))]
        merged_tuples.sort(key=lambda x: x[1], reverse=True)
        return [name for name, height in merged_tuples]

File: problems/2502-sort-the-people/sort-the-people.js
/**
 * @param {string[]} names
 * @param {number[]} heights
 * @return {string[]}
 */
const sortPeople = (names, heights) => {
    const arr = []
    for (let i = 0; i < names.length; i++) {
        arr[i] = [names[i], heights[i]];
    };
    return arr.sort((a,b) => b[1] - a[1]).map(e => e[0])
};